rskj-core/src/test/java/co/rsk/peg/P2shP2wshErpFederationNewRedeemTest.java

```diff
@@ -0,0 +1,273 @@
+/*
```

Comment:
Should this be all commented out?

Reply:
i actually deleted this and other test related to segwit (not the ReleaseWitnessTransactionBuilderTest). they are now included in bitcoinj-thin final-segwit-poc branch!